data/src/main/java/com/microsoft/azure/kusto/data/AadAuthenticationHelper.java

```diff
@@ -197,7 +197,7 @@ private AuthenticationResult acquireAccessTokenUsingDeviceCodeFlow() throws Exce
 
     private AuthenticationResult waitAndAcquireTokenByDeviceCode(DeviceCode deviceCode, AuthenticationContext context)
             throws InterruptedException {
-        int timeout = 20 * 1000;
+        int timeout = 60 * 1000;
```

Comment:
I think it's a bit too much

Reply:
it is a race to make it work - its not the first time i see people struggle to make it in 20 seconds ..  
how much do you think

Reply:
Fair enough